delphin/cli/edm.py

```diff
@@ -47,6 +50,17 @@ def call_compute(args):
     print(f'  F-score:\t{f}')
 
 
+def eds_from_mrs(m: mrs.MRS, predicate_modifiers=True, skip_on_errors=True) -> eds.EDS:
```

Comment:
I see you've added the `predicate_modifiers` option and changed the `skip_on_errors` default to `True`. This is fine, but there need to be corresponding ways to configure them. If `predicate_modifiers` is going to be optional, then there should be a `--no-predicate-modifiers` (or similar) option, otherwise the parameter can be removed. For `skip_on_errors`, if the default is `True` then the option could be `--stop-on-errors` or `--no-skip-errors` or something.